lang/cs/Org.Apache.REEF.Tests/Functional/FaultTolerant/TestResubmitContext.cs

```diff
@@ -0,0 +1,235 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using Org.Apache.REEF.Common.Context;
+using Org.Apache.REEF.Common.Tasks;
+using Org.Apache.REEF.Common.Tasks.Events;
+using Org.Apache.REEF.Driver;
+using Org.Apache.REEF.Driver.Context;
+using Org.Apache.REEF.Driver.Evaluator;
+using Org.Apache.REEF.Driver.Task;
+using Org.Apache.REEF.Tang.Annotations;
+using Org.Apache.REEF.Tang.Interface;
+using Org.Apache.REEF.Tang.Util;
+using Org.Apache.REEF.Utilities;
+using Org.Apache.REEF.Utilities.Logging;
+using Xunit;
+
+namespace Org.Apache.REEF.Tests.Functional.FaultTolerant
+{
+    [Collection("FunctionalTests")]
+    public sealed class TestResubmitContext : ReefFunctionalTest
+    {
+        private const string FailedEvaluatorMessage = "FailedEvaluatorMessage";
+        private const string CompletedTaskValidationMessage = "CompletedTaskValidationmessage";
+        private const string CompletedEvaluatorValidationMessage = "CompletedEvaluatorValidationMessage";
+        private const string FailSignal = "Fail";
+        private const string SuccSignal = "Succ";
+        private const string TaskId = "TaskId";
+
+        [Fact]
+        [Trait("Priority", "1")]
+        [Trait("Category", "FunctionalGated")]
+        [Trait("Description", "Test invocation of ResubmitEvaluatorDriver")]
+        public void ResubmitContextOnLocalRuntime()
```

Comment:
Please also add some documentation as to what is being tested, e.g. what bugs would be detected here.